orchestrator-salt/src/main/resources/salt/salt/postgresql/disaster_recovery/scripts/backup_db.sh

```diff
@@ -81,6 +82,8 @@ run_kinit() {
     doLog "Successful kinit using solr principal"
   elif kinit_as atlas "$ATLAS_KEYTAB"; then
     doLog "Successful kinit using atlas principal"
+  elif kinit_as knox "$KNOX_KEYTAB"; then
+    doLog "Successful kinit using knox principal"
```

Comment:
Should we have some final `else` statement that lists the users we failed to kinit as? For troubleshooting if we ever need it.

Reply:
The logging done by the **kinit** function takes care of this. When you run the scripts the logs will tell you exactly which kinit failed and what the actual principal used for authentication was.